### DISCUSSION TITLE
Delete Node / Subtree

### DISCUSSION THREAD

**Discussion by convolutionbbs** [kayjan/bigtree#299] [Questions] [answered]
Hello,
Is it possible to delete a node and all the children under it (a subtree)?  All I could find in the documentation was prune_tree, which sort of does the opposite.

Thanks!

### Description

delete a node and all children from a tree

**Comment by kayjan** [comment, marked as answer]
Hello, you can assign the children of that Node to be `None`, that should remove all descendants of the node. Is it what you’re intending to do?
